run.py

```diff
@@ -24,6 +24,172 @@
 from WDL.Type import Base as WDLBase
 
 
+def get_first_wdl_line(filename: str) -> str:
+    """
+    Get the first line of code (not a comment or whitespace) in a wdl file
+    """
+    with open(filename, 'r') as f:
+        for line in f.readlines():
+            # skip over comments and empty lines
+            if line.lstrip().startswith("#"):
+                continue
+            if line.strip() == '':
+                continue
+
+            return line
+
+
+def get_wdl_version_from_file(filename: str) -> str:
+    """
+    Find the wdl version of a wdl file through parsing
+    """
+    line = get_first_wdl_line(filename)
+
+    # get version
+    if "version 1.0" in line:
+        return "1.0"
+    elif "version 1.1" in line:
+        return "1.1"
+    else:
+        return "draft-2"
+
+
+def generate_change_command_string(lines):
+    """
+    Generator to change the expression placeholder syntax in command strings for draft-2
+
+    ex:
+    command {
+        ~{var}
+    }
+    turns into
+    command {
+        ${var}
+    }
+
+    Syntax must follow the above and isolated closing braces should only indicate where the command string ends
+
+    """
+    iterator = iter(lines)
+    in_command = False
+    for line in iterator:
+        if in_command is False:
+            if line.strip() == "command <<<" or line.strip() == "command {":
+                in_command = True
+            yield line
+        else:
+            if line.strip() == ">>>" or line.strip() == "}":  # this could be accidentally triggered
```

Comment:
This probably will get us in trouble at some point.

In fact I think it will get us in trouble with your other PR, where you define a Bash function in a command section and end it with } on a line.

We could say that for our source WDLs we need to use `<<< >>>` syntax always, for ease of parsing? Though then it's hard to test for support of `{}` syntax.